assets/home/sections/sensei-pro-ad.js

```diff
@@ -104,7 +104,7 @@ const SenseiProAd = ( { show } ) => {
 							</ul>
 
 							<a
-								href="https://senseilms.com/checkout/?add-to-cart=7009"
+								href="https://senseilms.com/checkout/?add-to-cart=7009&utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=sensei_home"
```

Comment:
Since we created a function to add these query strings, shouldn't we use it here? My thought is because if we change that we'd probably expect to change all the cases. Does it make sense for you?

Reply:
You are completely right! Done in 8d72c704e